src/ray/object_manager/pull_manager.h

```diff
@@ -170,6 +170,10 @@ class PullManager {
 
   void SetOutOfDisk(const ObjectID &object_id);
 
+  int64_t NumInactivePulls(const std::string &task_name) const {
+    return task_argument_bundles_.inactive_by_name.Get(task_name);
```

Comment:
Forgot the threading model here... this arg is not used other than the main thread right?

Reply:
I believe pull_manager and dependency_manager are on the same main event loop.